wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -270,24 +270,63 @@ A notice should be placed where appropriate in a section, but must start off the
 - `Caution` should be used to warn the reader to avoid unintended consequences.
 - `Warning` should be used to warn the reader that action may be taken against them.
 
+## Stacked notices and hatnotes
+
+Multiple notices and hatnotes may be stacked when necessary. When doing this, they must be stacked without blank lines and use trailing backslashes:
+
+```markdown
+*Warning: {warning}.*\
+*See also: {article}*
+```
+
+In many cases, it may be more fitting to embed extraneous notices or hatnotes into paragraph text instead of stacking many of them.
+
 ## Emphasising
 
-### Bold
+### Bolding
 
-Bold must use double asterisks (`**`).
+Bold text must use double asterisks (`**`).
 
 Lead paragraphs may bold the first occurrence of the article's title.
 
 ### Italics
 
 Italics must use single asterisks (`*`).
 
-Names of work or video games should be italicised. osu!—the game—is exempt from this.
-
 The first occurrence of an abbreviation, acronym, or initialism may be italicised.
 
 Italics may also be used to provide emphasis or help with readability.
 
+Names of work or video games should be italicised. osu!—the game—is exempt from this.
+
+As an example, when referring to songs in the format of `{artist} - {title}`, the whole part is a reference to the work and should therefore be italicised:
+
+```markdown
+*cYsmix - triangles* is a one of the three intro songs that can be heard when starting the game client.
+```
+
+Artist names are otherwise generally not italicised. This means that in free-form references, only the title should be italicised, because the artist name is then not part of the name of the work:
+
+```markdown
+*Blue Zenith* by xi is an infamous song in the osu! community due to a famous score set by a top player on a beatmap of the song.
```

Comment:
i'm not sure what you mean here. even if i was more specific without linking directly (for clarity of the example) it would be "Asphyxia's beatmap of the song". here at osu, people create beatmaps of songs, don't they?

Reply:
just "beatmap", or "a corresponding beatmap", would be enough imo

Reply:
```suggestion
*Blue Zenith* by xi is an infamous song in the osu! community due to a famous score set by a top player on a certain beatmap.
```